src/app/cli.sh

```diff
@@ -108,7 +108,11 @@ cli::custom() {
 		return 1
 	fi
 
-	drivers::$driver::$sub_cmd || return 1
+	# remove first arg 'custom' from $@ to capture additional args
+	shift
```

Comment:
make this more explicit, like in `main.sh`:
https://github.com/benschw/horde/blob/master/src/app/main.sh#L61